src/schemas/office-address-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { required, maxLength } from 'vuelidate/lib/validators'
+// The Address schema containing Vuelidate rules.
+// NB: This should match the subject JSON schema.
+export const officeAddressSchema = {
+  streetAddress: {
+    required,
+    maxLength: maxLength(50)
+  },
+  streetAddressAdditional: {
+    maxLength: maxLength(50)
+  },
+  addressCity: {
+    required,
+    maxLength: maxLength(40)
+  },
+  addressCountry: {
+    required,
+    // FUTURE: create new validation function isCountry('CA')
+    isCanada: (val:string) => Boolean(val === 'CA')
```

Comment:
Please follow up on whether this rule (and the region rule below) are needed here. If you're not sure, just comment it out for now and add a TODO comment :)

Reply:
I believe Sienna confirmed this today when i inquired about the validators on the api, that the BC and Canada rules will apply to our addresses in Incorporation filings similar to Change of address filings.So I believe the backend will have a validator on the api checking both, as far as i know.

Reply:
The FE needs validation to warn the user up-front (user-friendly / prevent errors) and the BE needs validation to protect itself from bad data.

I suspect we will need eventually need 2 address schemas -- one for business address(es) and one for director address(es) -- same as coops-ui.

Here, just handle the current case (office).

Reply:
Sienna confirmed that the address rules are the same as in coops. So no change here